shared/oae/macros/list.html

```diff
@@ -200,13 +200,14 @@
- * @param  {Boolean}                                        [displayOptions.largeDefault]          Whether or not a large default thumbnail icon should used. By default, a small icon will be used
- * @param  {Boolean}                                        [displayOptions.addLink]               Whether or not a link to the entity profile should be placed on the image. By default, a link will be set
- * @param  {String}                                         [displayOptions.customImage]           URL to the custom entity profile picture that should be used. If not provided, this will fall back to the thumbnailUrl
- * @param  {String}                                         [displayOptions.linkTarget]            The target of the thumbnail link. One of `_blank`, `_self`, `_parent` or `_top`. By default, the link will open in the same window
+ * @param  {User|Group|Content|Folder|Discussion|String}    entityData                          Standard object representing an OAE entity or a search result for an OAE entity. Alternatively, a string representing the resourceType or resourceSubType (i.e., 'user', 'group', 'content', `folder`, 'file', 'link', 'collabdoc', `discussions`) can be passed in for an empty/anonymous entity thumbnail
+ * @param  {Object}                                         [displayOptions]                    Display options for the thumbnail
+ * @param  {Boolean}                                        [displayOptions.addLink]            Whether or not a link to the entity profile should be placed on the image. By default, a link will be set
+ * @param  {Boolean}                                        [displayOptions.addVisibilityIcon]  Whether or not the visibility icon should be shown, this does not apply to users who never have a visibility icon. By default, a visibility icon is shown for all resources other than users
+ * @param  {String}                                         [displayOptions.customImage]        URL to the custom entity profile picture that should be used. If not provided, this will fall back to the thumbnailUrl
+ * @param  {String}                                         [displayOptions.flyover]            The content to put in a fly-over when the mouse hovers over the thumbnail image
+ * @param  {String}                                         [displayOptions.linkTarget]         The target of the thumbnail link. One of `_blank`, `_self`, `_parent` or `_top`. By default, the link will open in the same window
+ * @param  {String}                                         [displayOptions.size]               If specified, this content will suffix a class `oae-thumbnail-<size>`. Core CSS has support for "medium" and "large", and by default the `oae-thumbnail` is styled with what we would consider "small". "large" is recommend for thumbnails larger than 100px, while "medium" is recommended for larger than 50px
```

Comment:
You're using `small` above, so it should at least be specified as an expected option